tests/module/test_stdin2.js

```diff
@@ -0,0 +1,18179 @@
+// The Module object: Our interface to the outside world. We import
```

Comment:
Adding the file `tests/module/test_stdin2.js` into the PR looks accidental, this file seems to be a compiled output code from Emscripten, rather than a source file? Can you rewrite the commit history to remove this file?

Reply:
I've deleted it - I'm a little bit nervous rewriting the history since I read somewhere the Github can lose comments it you change the commit guids.

If you do a 'squeeze' when this is merged though that should take of it, I think.